### PR TITLE
v4.0.0-alpha.27 

### PR DESCRIPTION
**Pull request by dtassone**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,92 @@
...
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.27](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.26...v4.0.0-alpha.27)
+
+_Apr 29, 2021_
+
+Big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Add getRowClassName prop (#1448) @m4theushw
+- ⚡️ Update to Node 12 (#1499) @ZeeshanTamboli
```
Comment (oliviertassinari) [inline CHANGELOG.md:13]: I think that framing the information in this direction would have more utility. 

```suggestion
- ⚡️ Drop support for Node v10(#1499) @ZeeshanTamboli
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,92 @@
...
+- ♿ Make checkbox focusable (#1421) @dtassone
+- 🇮🇹 Add itIT locale (#1446) @profcav
+- 🇷🇺 Add ruRU locale (#1449) @Lukin
+- 🐞 Bugfixes
+
+### @material-ui/x-grid@v4.0.0-alpha.27 / @material-ui/data-grid@v4.0.0-alpha.27
+
+#### Breaking changes
+
+- [DataGrid] Remove `onAction` APIs (#1453) @DanailH
```
Comment (oliviertassinari) [inline CHANGELOG.md:23]: This is not impacting the MIT version

```suggestion
- [XGrid] Remove `onAction` APIs (#1453) @DanailH
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,92 @@
...
+
+This change has involved the following refactorings.
+
+  - Changes on `apiRef.current`.
```
Comment (oliviertassinari) [inline CHANGELOG.md:27]: A bullet point doesn't make sense in its context, it's alone.

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,92 @@
...
+-  onResize
+-  onUnmount
+-  onRowSelected
+-  onSelectionModelChange
+-  onSortModelChange
+-  onStateChange
+```
+Note: These methods are available as React props.
+
+- [DataGrid] Refactor useGridColumnResize (#1380) @DanailH
```
Comment (oliviertassinari) [inline CHANGELOG.md:42]: Same as above 

```suggestion
- [XGrid] Refactor useGridColumnResize (#1380) @DanailH
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,92 @@
...
+
+  - Changes on `apiRef.current`.
```
Comment (oliviertassinari) [inline CHANGELOG.md:44]: Same as above. 

```suggestion
 Changes on `apiRef.current`.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,92 @@
...
+- [DataGrid] Add getRowClassName prop (#1448) @m4theushw
+- [DataGrid] Add support for `classes` prop (#1450) @ZeeshanTamboli
+- [DataGrid] Allow to customize the overlay when there're no filtered rows (#1445) @m4theushw
+- [DataGrid] Correct quantities plPL (#1487) @Chriserus
+- [DataGrid] Fix autoPageSize with small dataset (#1505) @dtassone
+- [DataGrid] Fix delete key for uneditable cells (#1497) @dtassone
+- [DataGrid] Fix invalid translation key (#1504) @DanailH
+- [DataGrid] Forward props for all Toolbar and Footer components (#1456) @DanailH
+- [DataGrid] Improve support of core v5 (#1458) @oliviertassinari
+- [DataGrid] refactoring on focus cell and cols (#1421) @dtassone
```
Comment (oliviertassinari) [inline CHANGELOG.md:63]:
```suggestion
- [DataGrid] Improve focus management logic (#1421) @dtassone
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,92 @@
...
+- [core] upgrade node version from 10 to 12 (#1499) @ZeeshanTamboli
+
+#### Changes
+
+- [core] Label our packages as side effect free (#1466) @oliviertassinari
+- [core] Reduce work in data grid (#1520) @oliviertassinari
+- [core] Remove React.FC (#1436) @ZeeshanTamboli
+- [license] No need to test the location (#1488) @oliviertassinari
+  
+### Tests
```
Comment (oliviertassinari) [inline CHANGELOG.md:86]: We usual group tests under `### Core` as the second is about the underlying infrastructure that supports the components.

File: CHANGELOG.md [hunk 8]
```diff
@@ -3,6 +3,92 @@
...
+- [docs] Better document how to disable row selection (#1510) @ZeeshanTamboli
+- [docs] Fix data grid feature comparison (#1516) @imsuvesh
+- [docs] Fix typos (#1447) @ZeeshanTamboli
+- [docs] No ads for commercial license (#1489) @oliviertassinari
+
+### Core
+
+#### Breaking changes
+
+- [core] upgrade node version from 10 to 12 (#1499) @ZeeshanTamboli
```
Comment (oliviertassinari) [inline CHANGELOG.md:77]: I think that it should be listed under the header for the main packages. It impacts developers directly. Core changes are often second order consequences.

**Review by oliviertassinari — CHANGES_REQUESTED** [review]
We miss the update of the version of the packages.

**Comment by dtassone** [comment]
> We miss the update of the version of the packages.

I think it was easier and less error-prone when it was done automatically. Here you can easily miss the dependencies.

**Comment by dtassone** [comment]
> > We miss the update of the version of the packages.
> 
> I think it was easier and less error-prone when it was done automatically. Here you can easily miss the dependencies.

That's because I've done it manually.

**Comment by oliviertassinari** [comment]
> That's because I've done it manually.

I don't understand, [the instructions](https://github.com/mui-org/material-ui-x/blob/master/scripts/README.md#prepare asks for using `yarn release:version`, which does the update automatically.

**Review by oliviertassinari — APPROVED** [review]
Ok, I can't notice anything wrong, I'm in

github com_mui-org_material-ui-x_releases_edit_v4 0 0-alpha 26 (1)

File: CHANGELOG.md [hunk 9]
```diff
@@ -3,6 +3,85 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.27](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.26...v4.0.0-alpha.27)
+
+_Apr 29, 2021_
```
Comment (oliviertassinari) [inline CHANGELOG.md:8]:
```suggestion
_Apr 30, 2021_
```

**Comment by dtassone** [comment]
Great!!

**Pull request merged**